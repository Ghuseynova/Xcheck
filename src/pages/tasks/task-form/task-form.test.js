import React from 'react';
import { shallow } from 'enzyme';
import TaskForm from './task-form';

describe('render ', () => {
  test('Task form', () => {
    const tree = shallow(<TaskForm/>);
	  expect(tree).toMatchSnapshot();
	});
});