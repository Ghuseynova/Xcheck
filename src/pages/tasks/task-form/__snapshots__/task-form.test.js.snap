// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render  Task form 1`] = `ShallowWrapper {}`;
